ui/app/components/job-status/deployment-history.js

```diff
@@ -5,9 +5,13 @@ import { tracked } from '@glimmer/tracking';
 import { inject as service } from '@ember/service';
 import { action } from '@ember/object';
 
+const MAX_NUMBER_OF_EVENTS = 500;
+
 export default class JobStatusDeploymentHistoryComponent extends Component {
   @service notifications;
 
+  @tracked isHidden = this.args.isHidden;
```

Comment:
`note`: If you ever invoke the component with the value of this argument as a variable instead of a hard coded boolean value, this property will not reactively update based on the value of that variable changing. Using a [reactive getter](https://github.com/hashicorp/design-system/blob/main/packages/components/addon/components/hds/button/index.js#L92) will remove the possibility of a bug like this.